cmd/exo/kill.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"github.com/deref/exo/internal/core/api"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(killCmd)
+	killCmd.Flags().StringVarP(&killFlags.Signal, "signal", "s", "kill", "")
+}
+
+var killFlags struct {
+	Signal string
+}
+
+var killCmd = &cobra.Command{
+	Use:   "kill [ref...]",
+	Short: "Force stop processes",
+	Long: `Sends a unix signal to process-like components.
+	
+If no refs are provided, signals all processes in the workspace.
+	
+The default signal is SIGKILL.`,
```

Comment:
SIGKILL being the default feels strange when other kill commands generally use SIGTERM as the default.

Reply:
Understood, but 1) `stop` already does sigterm, then timeout, then sigkill. 2) 90% of the time when I use unix `kill`, I use `kill -9` and 3) this is the default in docker compose.

Reply:
:+1: Persuaded